app/client/src/config/cyanMetadata.ts

```diff
@@ -0,0 +1,36 @@
+export default [
```

Comment:
These are the column headers for the data fetched from the `cellConcentration` service (with indices 0, 254, and 255 removed, since they represent non-detection): each column represents a cell concentration range, and each value of the column represents a count of the number of pixels that fall in the range on a particular day.